src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -53,4 +53,24 @@
         public static final String CREDIT = "CR";
         public static final String DEBIT = "DR";
     }
+    
+    public class StandardAccountingExtractPdpConstants {
+        public static final String PAYMENT_CODE_CASH = "CASH";
+        public static final String COMBINDED_GROUP_INDICATOR = "Y";
+        public static final String BANK_CODE = "DISB";
+        public static final String FS_ORIGIN_CODE = "Z6";
+        public static final String DOCUMENT_TYPE = "APTR";
+        public static final String EMPLOYEE_STATUS_CODE = "EMPLOYEE";
+        public static final String EMPLOYEE_PAYEE_STATUS_TYPE_CODE = "E";
+        public static final String NON_EMPLOYEE_PAYEE_STATUS_TYPE_CODE = "Y"; 
+        public static final String DEFAULT_CHART_CODE = "IT";
+        public static final String DEFAULT_UNIT = "CRNL";
+        public static final String DEFAULT_SUB_UNIT = "CNCR";
+        public static final String FEED_FILE_ENTRY_HEADER_VERSION = "1.0";
+    }
+    
+    public static final String DATE_FORMAT = "MM/dd/YYYY";
+    public static final String EMPLOYEE_GROUP_ID = "CORNELL";
```

Comment:
My requirement was that the eighth element in the SAE must be Cornell, otherwise it is an error.  I don't have a strong opinion on what to call it.